src/json-formatter.js

```diff
@@ -143,6 +143,9 @@ angular.module('jsonFormatter', ['RecursionHelper'])
         scope.isOpen && !scope.isArray();
     };
 
+    if (!!attrs.title) {
```

Comment:
but if adding a new function getDisplayName, I can move this filter into the method to ignore nonsense title value(when !!title is false)